google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineFacet.java

```diff
@@ -38,6 +40,8 @@ public AppEngineFacet(@NotNull FacetType facetType,
       @NotNull String name,
       @NotNull AppEngineFacetConfiguration configuration) {
     super(facetType, module, name, configuration, null);
+
+    UsageTrackerProvider.getInstance().trackEvent(GctTracking.APP_ENGINE_CREATE_FACET).ping();
```

Comment:
There are two ways of adding a facet to a project, after it's created: Project Settings, Facets and add there or the Configure popup.

We control the Facet UI (AppEngineFacetEditor), so we're using onFacetInitialized to track facets added through Project Settings.

The only place we're able to detect facets added through the Configure popup is in the constructor. However, a constructor ping would also capture all other ways of a facet being created, including invalid ones (e.g., when the IDE is started), which is why I'm tracking this here.